test/test_calendar.py

```diff
@@ -101,3 +107,17 @@ def test_schedule_read_all(client, vcr_cassette):
     assert isinstance(schedules[0], Schedule)
 
     assert len(vcr_cassette) == 1
+
```

Comment:
You can delete the get_all test and just have a test for the WorkerHours get, WorkHours create, and update